tests/retrieve/test_echo_class.py

```diff
@@ -144,26 +144,28 @@ def test_hydroclass_semisupervised():
 
     mass_centers = pyart.retrieve.echo_class._get_mass_centers(10e9)
 
-    hydro_nofreq = pyart.retrieve.hydroclass_semisupervised(radar_small)
+    hydro_nofreq = pyart.retrieve.hydroclass_semisupervised(radar_small)["hydro"]
+
     assert "units" in hydro_nofreq.keys()
     assert "standard_name" in hydro_nofreq.keys()
     assert "long_name" in hydro_nofreq.keys()
     assert "coordinates" in hydro_nofreq.keys()
 
-    assert hydro_nofreq["data"].dtype == "int64"
-    assert_allclose(hydro_nofreq["data"][0][0:5], [6, 6, 6, 6, 6])
-    assert_allclose(hydro_nofreq["data"][0][-5:], [2, 2, 2, 2, 2])
+    assert hydro_nofreq["data"].dtype == "uint8"
+    assert_allclose(hydro_nofreq["data"][0][0:5], [7, 7, 7, 7, 7])
+    assert_allclose(hydro_nofreq["data"][0][-5:], [3, 3, 3, 3, 3])
 
     radar_small.instrument_parameters["frequency"] = {"data": np.array([5e9])}
-    hydro_freq = pyart.retrieve.hydroclass_semisupervised(radar_small)
-    assert_allclose(hydro_freq["data"][0][0:5], [6, 6, 6, 6, 6])
-    assert_allclose(hydro_freq["data"][0][-5:], [2, 2, 2, 2, 2])
+    hydro_freq = pyart.retrieve.hydroclass_semisupervised(radar_small)["hydro"]
+    assert_allclose(hydro_freq["data"][0][0:5], [7, 7, 7, 7, 7])
+    assert_allclose(hydro_freq["data"][0][-5:], [3, 3, 3, 3, 3])
 
     hydro = pyart.retrieve.hydroclass_semisupervised(
         radar_small, mass_centers=mass_centers
-    )
-    assert_allclose(hydro["data"][0][0:5], [6, 6, 6, 6, 6])
-    assert_allclose(hydro["data"][0][-5:], [2, 2, 2, 2, 2])
+    )["hydro"]
+
+    assert_allclose(hydro["data"][0][0:5], [7, 7, 7, 7, 7])
+    assert_allclose(hydro["data"][0][-5:], [3, 3, 3, 3, 3])
 
```

Comment:
Are we able to test the two other types of dicts as well? Propx and entropy?

Reply:
Hi @zssherman ,
Thanks for the review!

I have added additional tests for all functions and for the additional outputs of hydroclass_semisupervised

Reply:
@wolfidan Awesome thank you!